src/app/view/ChessBoardView.ts

```diff
@@ -19,3 +20,47 @@ export interface ChessBoardView {
   //showAvailableMoves
   //showSelectedPiece
 }
+*/
+
+export class Square {
+  constructor(private position_x: number, private position_y: number, private color: SquareColor) {}
+
+  setHtmlElement(): HTMLElement {
+    const position = this.mapSquarePosition();
+    const squareElement = document.createElement('div');
+    squareElement.id = position;
+    squareElement.classList.add(`square--${this.color}`);
+    return squareElement;
```

Comment:
to można skrócić (niepotrzebne np. zmienna "position")

Dodatkowo nie nadawać tu koloru pola